cmd/core/cmd.go

```diff
@@ -1,4 +1,4 @@
-package core
+package cmdCore
```

Comment:
typically I would keep the package name matching the directory name any reason not to in this case?

Reply:
there is a conflict between proto package core and cmd package